pkg/configs/db/dbtest/unit.go

```diff
@@ -26,3 +29,48 @@ func Setup(t *testing.T) db.DB {
 func Cleanup(t *testing.T, database db.DB) {
 	require.NoError(t, database.Close())
 }
+
+func TestUserPasswordFromPasswordFile(t *testing.T) {
+
+	tempFile, _ := ioutil.TempFile("", "testpassword")
+	defer tempFile.Close()
+	defer os.Remove(tempFile.Name())
```

Comment:
defer functions have a specific order.
In this case we are removing and then closing. We want the opposite.
https://go.dev/tour/flowcontrol/13#:~:text=Deferred%20function%20calls%20are%20pushed,in%2Dfirst%2Dout%20order.